modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -0,0 +1,83 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
+)
+
+func (suite *KeeperTestSuite) TestTrySendTx() {
+	var (
+		path   *ibctesting.Path
+		msg    interface{}
+		portID string
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success", func() {
+				amount, _ := sdk.ParseCoinsNormalized("100stake")
+				interchainAccountAddr, _ := suite.chainB.GetSimApp().ICAKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID)
+				msg = &banktypes.MsgSend{FromAddress: interchainAccountAddr, ToAddress: suite.chainB.SenderAccount.GetAddress().String(), Amount: amount}
+				portID = path.EndpointA.ChannelConfig.PortID
```

Comment:
nit: set this as the default before calling `tc.malleate()` so that test cases must explicitly change it to make it fail (reduces code and easier to verify a test case)

Reply:
Done :)